ipatests/test_integration/test_installation_client.py

```diff
@@ -94,6 +94,16 @@ def test_client_install_with_krb5(self):
         ).encode() not in krb5_cfg
         tasks.uninstall_client(self.clients[0])
 
+    def test_check_ssh_service_is_activated(self):
+        """
+        This test checks that ssh service is
+        activated in sssd.conf
+        """
+        tasks.install_client(self.master, self.clients[0])
+        sssd_cfg = self.clients[0].get_file_contents(paths.SSSD_CONF)
+        assert 'services = nss, pam, ssh, sudo' in sssd_cfg.decode()
```

Comment:
I'm a bit torn on this one because it actually checks that all default services are enabled and not just ssh. I feel pedantic for asking but maybe make this "check default sssd services are enabled" rather than ssh specifically?
I get why it as done this way because otherwise you'd have to parse the sssd config output to find the set of services and see if ssh is in there.
So I like the change, I'd just change the wording to check all default services with the inspiration to check that ssh is enabled.